interface/orders/procedure_stats.php

```diff
@@ -331,119 +331,114 @@ function process_result_code($row)
+                <?php
+                    foreach ($arr_by as $key => $value) {
+                        echo "    <option value='" . attr($key) . "'";
+                        if ($key == $form_by) {
+                            echo " selected";
+                        }
+
+                        echo ">" . text($value) . "</option>\n";
+                    }
+                ?>
+            </select>
+        </div>
+        <div class="col">
+            <?php echo xlt('Sex'); ?>:
+        </div>
+        <div class="col">
+            <select class='form-control' name='form_sexes' title='<?php echo xla('To filter by sex'); ?>'>
+                <?php
+                    foreach (array(3 => xl('Men and Women'), 1 => xl('Women Only'), 2 => xl('Men Only')) as $key => $value) {
+                        echo "       <option value='" . attr($key) . "'";
+                        if ($key == $form_sexes) {
+                            echo " selected";
+                        }
+
+                        echo ">" . text($value) . "</option>\n";
+                    }
+                ?>
+            </select>
+        </div>
+    </div>
+</div>
+<br>
+<div class="col-8 col-md-8">
+    <div class="row">
+        <div class="col">
+            <?php echo xlt('Columns'); ?>:
+            <select class='form-control' name='form_show[]' size='4' multiple title='<?php echo xla('Hold down Ctrl to select multiple items'); ?>'>
+                <?php
+                    foreach ($arr_show as $key => $value) {
+                        $title = $value['title'];
+                        if (empty($title) || $key == 'title') {
+                            $title = $value['description'];
+                        }
+
+                        echo "    <option value='" . attr($key) . "'";
+                        if (is_array($form_show) && in_array($key, $form_show)) {
+                            echo " selected";
+                        }
+
+                        echo ">" . text($title) . "</option>\n";
+                    }
+                ?>
+            </select>
+        </div>
+    </div>
+</div>
+<br>
+<div class="col-8 col-md-8">
+    <div class="row">
+        <div class="col">
+            <?php echo xlt('To{{Destination}}'); ?>:
+
+            <?php
+                foreach (array(1 => xl('Screen'), 2 => xl('Printer'), 3 => xl('Export File')) as $key => $value) {
+                    echo "   <input type='radio' name='form_output' value='" . attr($key) . "'";
+                    if ($key == $form_output) {
+                        echo ' checked';
+                    }
+
+                    echo " />" . text($value) . " &nbsp;";
+                }
+            ?>
+        </div>
+    </div>
+</div>
+<br>
+<div class="col-4 col-md-4">
+    <div class="col">
+         <button type='submit' class='btn btn-primary btn-save' name='form_submit' title='<?php echo xla('Click to generate the report'); ?>'><?php echo xla('Submit') ?></button>
```

Comment:
Finally was able to get the submit button to work. To get the submit button to work, it needs a value:
```
<button type='submit' class='btn btn-primary btn-save' name='form_submit' value='form_submit' title='<?php echo xla('Click to generate the report'); ?>'><?php echo xla('Submit') ?></button
```
If there is no value then it never passes the conditional on line 444 when submit the form

Reply:
After this is fixed, then will bring this PR into codebase.